ingestfile

```diff
@@ -68,28 +62,171 @@ _remove_mediaid_from_queue(){
+    fi
+}
+
+_delivertoia(){
+    if [ $(echo "${MEDIAID}" | grep -E "${REGEX4IA}") ] ; then
+        if [ $(which uploadia) ] ; then
+            _report -dt "Attempting to run uploadia and upload to Internet Archive"
+            uploadia "${OUTDIR_INGESTFILE}/${MEDIAID}/objects/access/youtube_up/${MEDIAID}.mp4" "${OUTDIR_INGESTFILE}/${MEDIAID}/objects/access/dvd/${MEDIAID}.iso"
+        fi
+    fi
+}
+
+_writecapturelog(){
+    echo -e "${LOG}" >> "${INGESTLOG}"
+    _writeingestlog "media ID" "${MEDIAID}"
+    _writeingestlog "media ID slate" "${MEDIAID4SLATE}"
+    _writeingestlog "operator" "${OP}"
+    _writeingestlog "input" "${INPUT}"
+    _writeingestlog "processing_directory" "${OUTDIR_INGESTFILE}"
+    _writeingestlog "audiodecision" "${AUDIODECISION}"
+    _writeingestlog "cropdecision" "${CROPDECISION}"
+    _writeingestlog "cleanupdecision" "${CLEANUPDECISION}"
+    _writeingestlog "formula" "${FORMULA}"
+    _writeingestlog "ingest_process_dir" "${OUTDIR_INGESTFILE}"
+    _writeingestlog "ingest_process_dir_freespace" "$(df -h "${OUTDIR_INGESTFILE}" | tail -n1 | awk '{print $4}')"
+    _writeingestlog "computer_name" "$(uname -n)"
+    _writeingestlog "user_name" "$(whoami)"
+    _writeingestlog "operating_system_version" "$(uname -v)"
+    _writeingestlog "datetime_start" "$(_get_iso8601)"
+    _writeingestlog "Series Title" "${SERIESTITLE}"
+    _writeingestlog "Episode Title" "${EPISODETITLE}"
+    _writeingestlog "made this" "ingestfile"
+    _writeingestlog "Interlacement test" "${INTERLACETEST}"
+    if [ -n "$(cat "${PHASEREPORT}" | xargs)" ] ; then
+    	_writeingestlog "Out_of_phase_audio_ranges" "$(cat "${PHASEREPORT}" | xargs)"
+    else
+        _writeingestlog "Out_of_phase_audio_ranges" "None"
+    fi
+    if [ "${INTIME}" ] ; then
+        _writeingestlog "intime" "${INTIME}"
+    fi
+    if [ "${OUTTIME}" ] ; then
+        _writeingestlog "outtime" "${OUTTIME}"
+    fi
+    if [ "${MODE}" == "preservation" ] ; then
+        _writeingestlog "package type" "Preservation-Video"
+    else
+        _writeingestlog "package type" "Production"
+    fi
+}
+
 _cleanup(){
-    _log -a "Process aborted"
+    _log -a "Process canceled"
     _remove_mediaid_from_queue
-    [ -d "${OUTDIR_INGESTFILE}/${MEDIAID}" ] && mv "${OUTDIR_INGESTFILE}/${MEDIAID}" "${OUTDIR_INGESTFILE}/${MEDIAID}_aborted_ingest_$(_get_iso8601_c)"
+    [ -d "${OUTDIR_INGESTFILE}/${MEDIAID}" ] && mv "${OUTDIR_INGESTFILE}/${MEDIAID}" "${OUTDIR_INGESTFILE}/${MEDIAID}_canceled_ingest_$(_get_iso8601_c)"
     exit 1
 }
 
+_cleanup_sourcefile(){
+    if [ -n "${RSYNC_ERR}" ] && [ "${RSYNC_ERR}" -gt "0" ] ; then
+        echo "${RSYNC_ERR}"
+        _report -wt "Cancelling requested deletion of source file, not looking safe."
+    else
+        _report -dt "Removing the source file from ${INPUT} as requested."
+        _run rm -f "${INPUT}"
+    fi
+}
+
 trap _cleanup SIGHUP SIGINT SIGTERM
 
 # check environment
-if [ -d "$(pwd)" ] ; then
-    echo ":)"
```

Comment:
Ha, thx for removing this reassuring smiley

Reply:
Sorry!! I figured it would be out of context for whoever's just watching the terminal but then again it's also a nice surprise